spec-main/fixtures/apps/background-color-transparent/renderer.js

```diff
@@ -0,0 +1,9 @@
+const { ipcRenderer } = require('electron');
+
+window.setTimeout(async (_) => {
+  document.body.style.background = 'transparent';
+}, 3000);
+
+window.setTimeout(async (_) => {
+  ipcRenderer.send('set-transparent');
+}, 5000);
```

Comment:
I'm aware this is somewhat brittle, but there's no real hook for validating that the color was successfully changed that i could find